<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoleSettingsDialog</class>
 <widget class="QDialog" name="HoleSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>271</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Maximum Y deviation</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Minimum circularity</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Maximum circularity</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Maximum CC deviation</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Minimum area</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Number of error frames</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QSpinBox" name="spnNErrorFrames"/>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spnMinArea">
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="dspnMaxYDev"/>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="dspnMinCircularity"/>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="dspnMaxCircularity"/>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="dspMaxCCDeviation"/>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="buttSaveClose">
     <property name="text">
      <string>Save &amp;&amp; close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dspnMaxYDev</tabstop>
  <tabstop>dspnMinCircularity</tabstop>
  <tabstop>dspnMaxCircularity</tabstop>
  <tabstop>dspMaxCCDeviation</tabstop>
  <tabstop>spnMinArea</tabstop>
  <tabstop>spnNErrorFrames</tabstop>
  <tabstop>buttSaveClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
